### PR TITLE
[UI] Add bottom navigation menu

### PR DESCRIPTION
**Pull request by Leo-Neat**
This PR adds a navigation menu to the bottom of the MCT Test page. This menu can be used in addition to swiping to support the legacy UI. It should make the UI more intuitive. Soon there will be a third option in addition to the  "Info" and "Test" pages which will contain test suites.

### CODE REVIEW

File: mediacontroller/src/main/res/menu/bottom_nav_menu.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (nic0lette) [inline mediacontroller/src/main/res/menu/bottom_nav_menu.xml:1]: nit: <!-- should be on the following line.

File: mediacontroller/src/main/res/layout/activity_media_app_testing.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-  Copyright (C) 2018 The Android Open Source Project
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (nic0lette) [inline mediacontroller/src/main/res/layout/activity_media_app_testing.xml:1]: nit: <!-- should be on the following line.

File: mediacontroller/src/main/res/drawable-anydpi/ic_test_suite.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (nic0lette) [inline mediacontroller/src/main/res/drawable-anydpi/ic_test_suite.xml:1]: Missing license header.

File: mediacontroller/src/main/res/drawable-anydpi/ic_test.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (nic0lette) [inline mediacontroller/src/main/res/drawable-anydpi/ic_test.xml:1]: Missing license header.

File: mediacontroller/src/main/res/drawable-anydpi/ic_info.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (nic0lette) [inline mediacontroller/src/main/res/drawable-anydpi/ic_info.xml:1]: Missing license header.

Comment (Leo-Neat) [reply mediacontroller/src/main/res/drawable-anydpi/ic_info.xml:13]: Done.

File: mediacontroller/src/main/res/drawable-anydpi/ic_test.xml (hunk 4 above)
Comment (Leo-Neat) [reply mediacontroller/src/main/res/drawable-anydpi/ic_test.xml:13]: Done.

File: mediacontroller/src/main/res/drawable-anydpi/ic_test_suite.xml (hunk 3 above)
Comment (Leo-Neat) [reply mediacontroller/src/main/res/drawable-anydpi/ic_test_suite.xml:13]: Done.

File: mediacontroller/src/main/res/layout/activity_media_app_testing.xml (hunk 2 above)
Comment (Leo-Neat) [reply mediacontroller/src/main/res/layout/activity_media_app_testing.xml]: Done.

File: mediacontroller/src/main/res/menu/bottom_nav_menu.xml (hunk 1 above)
Comment (Leo-Neat) [reply mediacontroller/src/main/res/menu/bottom_nav_menu.xml]: Done.

**Pull request merged**